modules/update-service-install-cli.adoc

```diff
@@ -0,0 +1,121 @@
+[id="update-service-install-cli_{context}"]
+= Installing the {product-title} Update Service by using the CLI
```

Comment:
The session is introducing the way to install the update service operator but the title is saying installing the Update Service. I would suggest to change it to Installing the {product-title} Update Service operator by using the CLI

Reply:
Agree. I've introduced the sections headers with either `Operator` or `application` to distinguish between operator and operand. This PR is obsolete and will be closed. Please see https://github.com/openshift/openshift-docs/pull/29630 for latest.